musicbot/bot.py

```diff
@@ -1317,18 +1317,33 @@ def _gen_embed(self):
                     reply=True, delete_after=10
                 )
 
-    async def cmd_id(self, author, user_mentions):
+    async def cmd_user(self, author, server, channel, message, user_mentions):
         """
         Usage:
-            {command_prefix}id [@user]
+            {command_prefix}user [@user]
 
-        Tells the user their id or the id of another user.
+        Provides some info on a user, if no-one mentioned returns info on message author.
         """
         if not user_mentions:
-            return Response(self.str.get('cmd-id-self', 'Your ID is `{0}`').format(author.id), reply=True, delete_after=35)
+            member = author
         else:
-            usr = user_mentions[0]
-            return Response(self.str.get('cmd-id-other', '**{0}**s ID is `{1}`').format(usr.name, usr.id), reply=True, delete_after=35)
+            member = user_mentions[0]
+
+        voice = member.voice.voice_channel
+        roles = ', '.join(r.name for r in member.roles)
+        join_date = member.joined_at
+        status = member.status
+        game = member.game
+        server = member.server
+        nickname = member.nick
+        color = member.colour
+        top_role = member.top_role
+        creation_date = member.created_at
+        avatar = member.avatar_url if member.avatar_url else member.default_avatar_url
+        user_id = member.id
+        return Response(self.str.get('cmd-user', """
```

Comment:
Please commit the updated `en.json` i18n file with the added string this references